frontend/packages/console-shared/src/components/status/Status.tsx

```diff
@@ -60,6 +71,9 @@ export const Status: React.FC<StatusProps> = ({ status, title, children, iconOnl
     case 'Up to date':
       return <SuccessStatus {...statusProps} />;
 
+    case 'Created':
+      return <StatusIconAndText {...statusProps} icon={<GreenCheckCircleIcon />} />;
```

Comment:
D'oh.  `case 'Created':` sholuld have been added on line 68.  Will fix in follow on.